irods_client_nfsrods/nfsrods_config/server.json

```diff
@@ -0,0 +1,21 @@
+{
+    "nfs_server": {
+        "port": 2049,
+        "irods_mount_point": "/tempZone",
+        "user_information_refresh_time_in_milliseconds": 3600000,
+        "file_information_refresh_time_in_milliseconds": 1000,
+        "user_access_refresh_time_in_milliseconds": 1000
+    },
+
+    "irods_client": {
+        "zone": "tempZone",
+        "host": "irods-catalog-provider",
+        "port": 1247,
+        "default_resource": "demoResc",
+        "ssl_negotiation_policy": "CS_NEG_REFUSE",
+        "proxy_admin_account": {
+            "username": "rods",
+            "password": "rods"
+        }
+    }
+}
```

Comment:
Is there a reason we don't define all config options?

Reply:
No reason. Should we?

Historical background: This was lifted from the OMERO experiment, which was lifted from... the original version of NFSRODS in this project, I believe.

Reply:
let's use a full example - best practice and all that.

Reply:
I figured that config was from an older version. I think including the full set of options is a good thing for the demo. It let's users see what's available.

Reply:
Done